Equilobe.DailyReport.Models/Dashboard/InstanceWorklogs.cs

```diff
@@ -0,0 +1,11 @@
+﻿using Equilobe.DailyReport.Models.Storage;
+using System.Collections.Generic;
+
+namespace Equilobe.DailyReport.Models.Dashboard
+{
+    public class InstanceWorklogs
```

Comment:
I don't think the class should be called InstanceWorklogs. I think AtlassianUserWorklogs would be better. A list of AtlassianUserWorklogs would be the instance worklogs.